### DISCUSSION TITLE
all etcd nodes are unavailable

### DISCUSSION THREAD

**Discussion by wwfengqi** [apache/apisix#8695] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/5087232/212803637-405c6e27-357d-4cc2-9341-0cce7576a964.png)

我是根据快速手册安装的，但是报错（ https://apisix.apache.org/zh/docs/apisix/getting-started/）
![image](https://user-images.githubusercontent.com/5087232/212803739-107beec5-d1f6-4761-9dad-6506d1c11cf5.png)

**Comment by tokers** [comment, marked as answer]
Did you see any related errors when you starting all the containers? It looks like the custom `apisix_conf/config.yaml` doesn't mount to APISIX: https://github.com/apache/apisix-docker/blob/master/example/apisix_conf/config.yaml.

**Reply by wwfengqi** [reply]
but i doesn't modify anything in fact.
![image](https://user-images.githubusercontent.com/5087232/212861343-524c0631-dd38-41d3-8ae1-cdc052949002.png)

**Reply by wwfengqi** [reply]
i exec the command of "chmod -R 777 ./example/" later,and it is Ok ,
